### PR TITLE
New CVE's - Security Bulletin

### PR DESCRIPTION
**Pull request by alagujeeva**
## Describe the Change

To include new CVE's for security bulletin

## Review Changes

💻 [Preview URL](https://deploy-preview-1510--docs-spectrocloud.netlify.app/security/security-bulletins/cve-reports)

🎫 [DOC-859](https://spectrocloud.atlassian.net/browse/DOC-859)

### CODE REVIEW

File: content/docs/09.5-security/20-security-bulletins/10-cve-reports.md [hunk 1]
```diff
@@ -96,6 +96,360 @@ Update cluster profiles using the vSphere CSI pack to version v2.6 or greater. A
...
+
+### Description
+
+Not all valid JavaScript whitespace characters are considered to be whitespace. Templates containing whitespace characters outside of the character set "\t\n\f\r\u0020\u2028\u2029" in JavaScript contexts that also contain actions may not be properly sanitized during execution.
+
+<br />
+
+### Impact
+
+No impact - We use html/template package but our html templates are static, and we don’t accept or parse any user given data.
```
[the remark this answers is not in the document]
Comment (alagujeeva) [reply content/docs/09.5-security/20-security-bulletins/10-cve-reports.md:111]: Yes

File: content/docs/09.5-security/20-security-bulletins/10-cve-reports.md [hunk 2]
```diff
@@ -96,6 +96,360 @@ Update cluster profiles using the vSphere CSI pack to version v2.6 or greater. A
...
+- https://nvd.nist.gov/vuln/detail/CVE-2023-24540
+- https://pkg.go.dev/vuln/GO-2023-1752
+
+### September 01, 2023 - CVE-2023-29402 - Go Vulnerability - Security Advisory - 9.8 CVSS
+
+<br />
+
+### Description:
+
+The go command may execute arbitrary code at build time when using cgo. This may occur when running "go get" on a malicious module, or when running any other command which builds untrusted code. This is can by triggered by linker flags, specified via a "#cgo LDFLAGS" directive. Flags containing embedded spaces are mishandled, allowing disallowed flags to be smuggled through the LDFLAGS sanitization by including them in the argument of another flag. This only affects usage of the gccgo compiler.
```
[the remark this answers is not in the document]
Comment (alagujeeva) [reply content/docs/09.5-security/20-security-bulletins/10-cve-reports.md:144]: Yes

[the remark this answers is not in the document]
Comment (alagujeeva) [reply content/docs/09.5-security/20-security-bulletins/10-cve-reports.md:144]: Yes

[the remark this answers is not in the document]
Comment (alagujeeva) [reply content/docs/09.5-security/20-security-bulletins/10-cve-reports.md:144]: Yes

[the remark this answers is not in the document]
Comment (alagujeeva) [reply content/docs/09.5-security/20-security-bulletins/10-cve-reports.md:144]: Yes

File: content/docs/09.5-security/20-security-bulletins/10-cve-reports.md [hunk 3]
```diff
@@ -96,6 +96,360 @@ Update cluster profiles using the vSphere CSI pack to version v2.6 or greater. A
...
+
+<br />
+
+### Description
+
+Templates do not properly consider backticks (`) as Javascript string delimiters, and do not escape them as expected. Backticks are used, since ES6, for JS template literals. If a template contains a Go template action within a Javascript template literal, the contents of the action can be used to terminate the literal, injecting arbitrary Javascript code into the Go template. As ES6 template literals are rather complex, and themselves can do string interpolation, the decision was made to simply disallow Go template actions from being used inside of them (e.g. "var a = {{.}}"), since there is no obviously safe way to allow this behavior. This takes the same approach as github.com/google/safehtml. With fix, Template.Parse returns an Error when it encounters templates like this, with an ErrorCode of value 12. This ErrorCode is currently unexported, but will be exported in the release of Go 1.21. Users who rely on the previous behavior can re-enable it using the GODEBUG flag jstmpllitinterp=1, with the caveat that backticks will now be escaped. This should be used with caution.
```
[the remark this answers is not in the document]
Comment (alagujeeva) [reply content/docs/09.5-security/20-security-bulletins/10-cve-reports.md:261]: Yes

**Review by afayasa — APPROVED** [review]
Thanks @karl-cardenas-coding. Looks good.

**Pull request merged**